### PR TITLE
remove mobile spacer from frontpage on desktop

### PR DESCRIPTION
**Pull request by Rowansdabomb**

### CODE REVIEW

File: ubyssey/templates/homepage/base.html [hunk 1]
```diff
@@ -4,7 +4,9 @@
 {% load dispatch_tags %}
 {% block content %}
 {% include 'headers/main.html' %}
-{% include 'headers/mobile.html' %}
+{% if is_mobile%}
```
Comment (JamieRL) [inline ubyssey/templates/homepage/base.html]: add space before last `%`

**Review by JamieRL — CHANGES_REQUESTED** [review]
just 1 thing. Looks good

File: ubyssey/templates/homepage/base.html (hunk 1 above)
Comment (Rowansdabomb) [reply ubyssey/templates/homepage/base.html]: nice catch

**Comment by Rowansdabomb** [comment]
fixed the spacing, should be ready to merge

**Review by JamieRL — APPROVED** [review]

**Pull request merged**